Modules/Loadable/Transforms/Documentation/Transforms.dox

```diff
@@ -2,7 +2,7 @@
 \defgroup Slicer_QtModules_Transforms Transforms
 \ingroup Slicer_QtModules
 This module is used for creating, editing, and displaying transformations.
-Transformation nodes are used in Slicer to define spacial relationships between different nodes (such as volumes, models, markups, ROI's, or other transform nodes) or between the nodes and the global RAS space.
+Transformation nodes are used in Slicer to define spatial relationships between different nodes (such as volumes, models, markups, ROI's, or other transform nodes) or between the nodes and the global RAS space.
```

Comment:
Here is another one. It may be misused in other places as well?
```suggestion
Transformation nodes are used in Slicer to define spatial relationships between different nodes (such as volumes, models, markups, ROIs, or other transform nodes) or between the nodes and the global RAS space.
```

Reply:
I found:

https://github.com/Slicer/Slicer/blob/571cba35f5715c344101b808c1283edec20adeb5/Modules/Loadable/CropVolume/Logic/vtkSlicerCropVolumeLogic.cxx#L676

but it seems correct in context, right?

Reply:
Yes that one is acceptable. Thanks for looking!